cocos/ui/UIEditBox/iOS/CCUISingleLineTextField.mm

```diff
@@ -45,4 +44,47 @@ - (CGRect)editingRectForBounds:(CGRect)bounds
     return [self textRectForBounds:bounds];
 }
 
+- (void)setInnerPadding:(float)left :(float)top :(float)right :(float)bottom
+{
+    auto glview = cocos2d::Director::getInstance()->getOpenGLView();
+    float factor = glview->getScaleX() / glview->getContentScaleFactor();
+    
+    _leftPadding = left * factor;
+    _topPadding = top * factor;
+    _rightPadding = right * factor;
+    _bottomPadding = bottom * factor;
+    
+}
+
+- (void)setTextAlignMent:(cocos2d::TextHAlignment)hAlign :(cocos2d::TextVAlignment)vAlign
```

Comment:
You can make your changes and commit, then push to the remote branch.
It's no need to open another PR.